test/pylib/suite/base.py

```diff
@@ -478,6 +483,7 @@ def report_error(error, failure_injection_desc = None):
                      # test for directory to store test temporary data.
                      TMPDIR=os.path.join(options.tmpdir, test.mode),
                      SCYLLA_TEST_ENV='yes',
+                     SCYLLA_TEST_RUNNER="test.py",
```

Comment:
As I commented above, I think this isn't needed if you need the special option "--test-py-init".